### PR TITLE
Sils/a

### PR DESCRIPTION
**Pull request by sils1297**

### CODE REVIEW

File: .codecov.yml [hunk 1]
```diff
@@ -1,12 +1,9 @@
...
-    status:
-      change:
-        enabled: false
-      patch:
-        enabled: false
-      project:
-        enabled: true
-        target: auto
-        threshold: 100%
+comment: false
```
Comment (Adrianzatreanu) [inline .codecov.yml]: would this make it stop commenting on github?

**Comment by Adrianzatreanu** [comment]
ack 4a7a994b

File: .codecov.yml (hunk 1 above)
Comment (sils1297) [inline .codecov.yml]: I hope, @stevepeak made this in a PR and he created codecov, let's give it a try?

**Comment by Adrianzatreanu** [comment]
ack f0b677a9

**Comment by Adrianzatreanu** [comment]
@rultor merge

**Pull request merged**